i18n/zh/docusaurus-plugin-content-docs/current/design-document/03-connect/10-file-connector.md

```diff
@@ -0,0 +1,46 @@
+# File
+## FileSinkConnector：从 EventMesh 写入文件 File
+1. 启动你的 EventMesh Runtime。
+2. 启用 sinkConnector 并检查 `sink-config.yml`。
```

Comment:
May you please add a new line between titles, lists, text and code blocks? It will improve the compatibility in different markdown viewers. ☺️

Reply:
ok

Reply:
I changed it，Thanks for the Pointers，and This is my other pr https://github.com/apache/eventmesh/issues/4781，Please give me your advice when you are free